fc-community-backend/apps/api/src/factchain-core/noteService.ts

```diff
@@ -64,6 +64,7 @@ export class NoteService {
 
   //Throw if doesn't exist
   getXNoteID = async (noteUrl: string) => {
+    const XUrl = noteUrl.replace("twitter", "x");
```

Comment:
let's replace in `createXNoteMetadata` as well, no?

Reply:
you're right